test/nameValidator.spec.ts

```diff
@@ -9,8 +9,8 @@ describe('NameValidator', () => {
 	});
 
 	it('should not map to an error message with empty or whitespace names', () => {
```

Comment:
should not map to an error message with empty or whitespace names ->
**should map** to an error message with empty or whitespace names 
or
`should not allow  empty or whitespace names`

Reply:
Resolved